### ISSUE TITLE
`build.sh` has system dependent conditional code

### ISSUE THREAD

**Issue by borupdaniel** [conda-forge/gpi-feedstock#16]
As is, it is contradictory to use `noarch` when the `build.sh` has system dependent conditional code like `if [ "$(uname)" ...`. Meaning, this shouldn't be `noarch` unless the build is changed to have consistent output supporting all platforms.

_Originally posted by @mbargull in https://github.com/conda-forge/gpi-feedstock/pull/13#issuecomment-562531025_

**Comment by borupdaniel** [comment]
Thanks for catching this, @mbargull. This is left over from before I took over the project and it looks like it's just a work-around for some differences in `cp`. I'm sure there's an OS-agnostic way to achieve the same thing.

**Comment by mbargull** [comment]
> differences in `cp`

No idea why those `cp -R` commands aren't done uniformly.

But the thing that is more important is the
```
    # ".command" script is required for terminal.app launcher
    cp launch/gpi.command $PREFIX/bin/
    # terminal.app launcher script, the target for GPI.app
    cp launch/gpi.app $PREFIX/bin/
```
vs
```
    LAUNCHER_PATH=$PREFIX/share/gpi
    mkdir -p $LAUNCHER_PATH
    cp launch/GPI.desktop $LAUNCHER_PATH/
    cp lib/gpi/graphics/iclogo.png $LAUNCHER_PATH/
```
part, which means the output was intentionally different on both platforms.

**Comment by borupdaniel** [comment]
I think that code is left over from previous attempts at packaging as something closer to a normal app (with an icon in the dock, etc.). I'm not supporting that at present, and when I get around to restoring it I think those steps should be implemented outside of the `noarch` package, anyhow, so I'm just going to comment these things out for the time being.

**Comment by borupdaniel** [comment]
Resolved by #18
